dev_scripts_helpers/github/sync_gh_issue_labels.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env python3
+"""
+Synchronize GitHub issue labels from a label inventory manifest file.
+
+Examples
+
+# Basic usage:
+> ./dev_scripts_helpers/github/sync_gh_issue_labels.py \
+    --input_file ./dev_scripts_helpers/github/labels/gh_issues_labels.yml \
+    --owner causify-ai \
+    --repo tutorials \
+    --token_env_var GITHUB_TOKEN \
+    --dry_run \
+    --backup
+"""
+
+import argparse
+import logging
+import os
+import subprocess
+from typing import Dict, List
+
+import yaml
+
+try:
+    import github
+except ModuleNotFoundError:
+    subprocess.call(["sudo", "/venv/bin/pip", "install", "pygithub"])
+    import github
+
+import helpers.hdbg as hdbg
+import helpers.hgit as hgit
+import helpers.hparser as hparser
+import helpers.hsystem as hsystem
+
+_LOG = logging.getLogger(__name__)
+
+
+# #############################################################################
+# Label
+# #############################################################################
+
+
+class Label:
+
+    def __init__(self, name: str, description: str, color: str):
+        """
+        Initialize the label with name, description, and color.
+
+        :param name: label name
+        :param description: label description
+        :param color: label color in hex format
+        """
+        self._name = name
+        self._description = description
+        # Remove '#' prefix from hex code if present.
+        self._color = color.lstrip("#")
+
+    def __repr__(self):
+        return f"label(name='{self.name}', description='{self.description}', color='{self.color}')"
+
+    @property
+    def name(self) -> str:
+        return self._name
+
+    @property
+    def description(self) -> str:
+        return self._description
+
+    @property
+    def color(self) -> str:
+        return self._color
+
+    def to_dict(self) -> Dict[str, str]:
+        """
+        Return label as a dictionary.
+
+        :return: label as a dictionary
+        """
+        return {
+            "name": self._name,
+            "description": self._description,
+            "color": self._color,
+        }
+
+
+def _load_labels(path: str) -> List[Label]:
+    """
+    Load labels from label inventory manifest file.
+
+    :param path: path to label inventory manifest file
+    :return: label objects
+    """
+    try:
```

Comment:
I'm not super fond of this approach of try ... except

My reasoning is that everything can throw exceptions so what are we really achieving? I use try ... except when I expected something could go wrong and I want to handle it to help the caller, since I know the inside of the code and thus I can make a better decision.
Here the only info is that we print an error. But is it worth it the visual noise?
My sensibility says it's now worth it.